Adds a page about accessibility to the designing bitcoin products section (#821)
BitcoinDesign/Guide: guide/designing-products/accessibility.md

## Why is it important?

As a payment network, bitcoin was designed to be borderless and permissionless. But if the applications people use to interact with the network are not versatile enough to accommodate diverse needs, then many of us will still not be able to take advantage of the benefits bitcoin provides. Products should be able to be used and accessed by all people, regardless of their social status, and so this should also extend into people with disabilities. [Developing nations](https://en.wikipedia.org/wiki/List_of_countries_by_literacy_rate) are the ones who seek to benefit the most from bitcoin. We want to improve usability for all people so they can easily navigate and move with ease through the applications. This means designing products that use plain language, support users with complex tasks, respect user preferences, work well with assistive technologies, and more.

A great side-effect is that a product designed with accessibility in mind is also easier to use for the part of the audience without impairments. This is referred to as the "[curb-cut](https://en.wikipedia.org/wiki/Curb_cut_effect)" effect. For example, many hearing people use closed captioning.

> As a wider community we also have a moral obligation to make products [inclusive]({{ '/guide/getting-started/principles/#inclusion' | relative_url }}) for everyone and to ensure that the experience they have with our product is suitable to them, regardless of their physical, mental abilities.

Reviewer: ```suggestion
As a wider community, we also have a moral obligation to make products [inclusive]({{ '/guide/getting-started/principles/#inclusion' | relative_url }}) for everyone and ensure that their experiences with our products are suitable, regardless of their physical and mental abilities.
```

## Categories of accessibility needs

#### Vision

> Full or partial blindness, color blindness and other visual disabilities. Three million people in the US alone have vision impairment and one million people are blind ([source](https://www.afb.org/research-and-initiatives/statistics)). See how a [blind person uses an iPhone](https://youtu.be/fjdcKioHb5w).

Reviewer: ```suggestion
This includes full or partial blindness, color blindness, and other visual disabilities. Three million people in the US alone have vision impairment, and one million people are blind ([source](https://www.afb.org/research-and-initiatives/statistics)). See how a [blind person uses an iPhone](https://youtu.be/fjdcKioHb5w).
```

#### Physical

> A physical disability can effect a person temporarily or permanently. On a practical level this affects a their ability to use a keyboard or mouse. Tapping a screen or holding a phone still to scan a QR code could pose as  a challenge. Seventy-five million people across the world need a wheel chair on a daily basis ([source](https://www.inclusivecitymaker.com/disabled-people-in-the-world-in-2021-facts-and-figures/)). Listen to a [quadriplegic talking about accessibility](https://www.youtube.com/watch?v=eMFrcJBX0pA).

Reviewer: ```suggestion
A physical disability can effect a person temporarily or permanently. On a practical level this affects their ability to use a keyboard or mouse. Tapping a screen or holding a phone still to scan a QR code could pose as  a challenge. Seventy-five million people across the world need a wheel chair on a daily basis ([source](https://www.inclusivecitymaker.com/disabled-people-in-the-world-in-2021-facts-and-figures/)). Listen to a [quadriplegic talking about accessibility](https://www.youtube.com/watch?v=eMFrcJBX0pA).
```

#### Intellectual

Learning difficulties refer to a lower ability to understand new or complex information. Someone with intellectual difficulties also struggles with learning and applying new skills and can also have difficulty reading or understanding things. Over two hundred million people have some form of intellectual disability ([source](https://www.inclusivecitymaker.com/disabled-people-in-the-world-in-2021-facts-and-figures/)).

#### Hearing

> Hearing difficulty such as being partly deaf, this can affect one or both ears. Any hearing loss below 20 decibels is considered a hearing loss. 400 million people have some form of deafness or hearing loss.([source](https://www.inclusivecitymaker.com/disabled-people-in-the-world-in-2021-facts-and-figures/))

Reviewer: ```suggestion
This includes hearing difficulties such as being completely or partly deaf, affecting one or both ears. Any hearing loss below 20 decibels is considered a hearing loss. 400 million people have some form of deafness or hearing loss ([source](https://www.inclusivecitymaker.com/disabled-people-in-the-world-in-2021-facts-and-figures/)).
```
I feel like a11y stuff for partly deaf people would apply just as easily to completely deaf people, so it seems strange to mention only partly deaf people specifically.

#### Literacy

> Illiteracy is the inability of a person to read/write or understand a simple statement that is related to his/her everyday life. There are over 700 million illiterate adults around the world and more than half of them are women. ([source](http://uis.unesco.org/en/topic/literacy))

Reviewer: ```suggestion
Illiteracy is a person's inability to read, write, or understand a simple statement that is related to their everyday life. There are over 700 million illiterate adults worldwide, and more than half of them are women ([source](http://uis.unesco.org/en/topic/literacy)).
```
Anti-slash coalition. Also, source links are included before sentence period in the above paragraphs.

## Tips and examples

Improving accessibility may seem like an overwhelming task. The most important thing is to get started, so let's take a look at some basics. Correctly labeling interactive elements is one of the essential practices in accessibility. It allows screen readers to understand and convey the information to impaired users in various ways. The following example illustrates some good and bad practices.

> {% include image-gallery.html pages = page.exampleImages %}

Reviewer: There's a ton of tiny detail in these images. This would be the perfect place to use the [modal images feature](https://bitcoin.design/guide/contribute/formatting/#image-viewable-in-larger-modal).

Reviewer 2: Good suggestion as indeed as there is alot of detail in the images, what are your thoughts @GBKS?

It is impossible to know what a button does when it is only labelled as "button", and not with its function. Testing has shown that QR codes are often not selectable at all, which makes them invisible to users reliant on screen readers. A good solution is a label such as "QR code to copy the receive address".

Also important is that elements are correctly ordered. Otherwise users reliant on the keyboard, or other linear navigation techniques, will not be able to easily understand what is on the screen.

> {% include image-gallery.html pages = page.exampleImagesTwo %}

Reviewer: There's a ton of tiny detail in these images. This would be the perfect place to use the [modal images feature](https://bitcoin.design/guide/contribute/formatting/#image-viewable-in-larger-modal).

Author: Modals are added.
